pyproject.toml

```diff
@@ -49,6 +49,7 @@ pytest-xdist = "^2.5.0"
 yanc = "^0.3.3"
 pyssim = "^0.4.0"
 scipy = "^1.7.3"
+mock = "^4.0.3"
```

Comment:
@victorjkhalaf `mock` is part of unittest: https://docs.python.org/3/library/unittest.mock.html#patch
There's no need to add it as a dependency.

Reply:
Strange, it had struggled to run at the time but I'll take it out!

Reply:
It was added on python 3.3, make sure that you're using at least Python 3.7.